recipe-server/client/control_new/components/recipes/Details.js

```diff
@@ -0,0 +1,48 @@
+import { Card, Tag } from 'antd';
+import { Map } from 'immutable';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+export default function Details({ recipe }) {
+  return (
+    <div>
+      <Card key="recipe-details" title="Recipe">
+        <dl className="details">
```

Comment:
This probably also needs to show the simple filter data, if it exists. I'm not sure if you want to do that in this PR or not though.